### DISCUSSION TITLE
How to disable vue feature in markdown

### DISCUSSION THREAD

**Discussion by liuliangsir** [vuejs/vitepress#1674] [Q&A]
As the title above says, I would like to disable all vue features inside all of markdown files since that default behavior(using vue in markdown) makes build process(`vitepress build docs`) failed with PLUGIN_ERROR code during transform hook. All markdown files are pure markdowns with only markdown own syntax and no Vue-specific syntax. There are massive stuffs with rewriting markdown file content for working around the error above. @brc-dd cc

**Comment by brc-dd** [comment]
You can try patching `dist/client/theme-default/components/VPDoc.vue` to add `v-pre` directive in `<Content />`.

**Reply by liuliangsir** [reply]
It stills to be not working and throws vite Internal server error(invalid end tag). In addition, These pages compiled by normal markdown files seem to be abnormal, whose contents are empty after adding `v-pre` directive in `<Content />`.

![image](https://user-images.githubusercontent.com/8049878/206179913-f20dfd47-ff85-416b-b640-b39c78ba13fb.png)

**Reply by brc-dd** [reply]
Ah, I'll take a look on this. Can you [create a feature request](https://github.com/vuejs/vitepress/issues/new?template=feature_request.yml) for this? It is easier to track things there.

**Reply by liuliangsir** [reply]
Okay
